pyuvdata/uvdata.py

```diff
@@ -3575,17 +3738,31 @@ def read(self, filename, axis=None, file_type=None,
             Option to check acceptable range of the values of parameters after
             reading in the file (the default is True, meaning the acceptable
             range check will be done). Ignored if read_data is False.
+
```

Comment:
I think we should add a `Raises` section. From a quick glance, it looks like we raise `ValueError` if we can't infer the file type, or if the user specifies both `antenna_nums` and `antenna_names`. (There may be more, I didn't do an exhaustive search.)

Reply:
I added a bunch of raises sections for the file type specific reads and writes throughout.